erpnext/stock/report/ordered_items_to_be_delivered/ordered_items_to_be_delivered.json

```diff
@@ -1,18 +1,18 @@
 {
- "add_total_row": 1, 
+ "add_total_row": 0, 
```

Comment:
@umaepoch Any specific reason for removing total row.

Reply:
I don't know how it got deleted Nabin. I have made the changes again, and am sending a pull request again for the same.